examples/subscribe_notify_characteristic.rs

```diff
@@ -0,0 +1,122 @@
+use std::time::Duration;
+
+use btleplug::api::{ValueNotification, CharPropFlags};
+use btleplug::api::{Central, Manager as _, Peripheral};
+use btleplug::platform::{Adapter, Manager};
+use std::io::Cursor;
+use std::error::Error;
+use uuid::Uuid;
+use tokio::time;
+use futures::stream::{StreamExt};
+
+const PERIPHERAL_NAME_MATCH_FILTER: &'static str = "Neuro"; // filter BLE device by partial name
+const NOTIFY_CHARACTERISTIC_UUID:Uuid = Uuid::from_u128(0x6e400002_b534_f393_67a9_e50e24dccA9e); // subscribe UUID
+
+#[cfg(target_os = "linux")]
+fn print_adapter_info(adapter: &Adapter) {
+    println!("connected adapter {:?}", adapter);
+}
+#[cfg(any(target_os = "windows", target_os = "macos"))]
+fn print_adapter_info(_adapter: &Adapter) {
+    println!("adapter info can't be printed on Windows 10 or mac");
+}
+
+/// Processing received BLE data
+fn my_on_notification_handler(data: ValueNotification) {
+    let rdr = Cursor::new(data.value);
+    println!("Received data from [{:?}] = {:?}", data.uuid, rdr);
+}
+
+/**
+If you are getting run time error like that :
+ thread 'main' panicked at 'Can't scan BLE adapter for connected devices...: PermissionDenied',
+ you can try to run app with > sudo ./discover_adapters_peripherals
+ on linux
+**/
```

Comment:
This comment is no longer relevant for the current Linux implementation, remove it.